src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -868,8 +610,7 @@ private static bool IsChunkedModeForSend(HttpRequestMessage requestMessage)
             return chunkedMode;
         }
 
-        // TODO: Refactor to smaller files. Perhaps split cookie handling into separate class.
-        private static string GetCookieHeader(Uri uri, CookieContainer cookies)
+        internal static string GetCookieHeader(Uri uri, CookieContainer cookies)
```

Comment:
Good idea. Longer term it should be an API we add to the CookieContainer class. 

For now, I will add as TODO to refactor to separate class.